flutter/mesh/lib/src/internal_stuff/tessellated_mesh.dart

```diff
@@ -28,96 +23,54 @@ class TessellatedMesh {
     required RenderedOMeshRect verticesMesh,
     required RenderedOMeshRect textureMesh,
     required int tessellation,
-    required bool impellerCompatibilityMode,
```

Comment:
We need to keep this on at least until we have a Flutter release with impeller working

Reply:
YEah makes sense, but while I was at it I couldn't help myself. :)